source/deployment/sso-saml-ldapsync.rst

```diff
@@ -29,3 +29,20 @@ Once the synchronization with AD/LDAP is enabled, user attributes are synchroniz
     
     If a user is deactivated from SAML, their session won't expire until they're deactivated from AD/LDAP. However, they won't be able to log back in to Mattermost.
  
+Technical description of SAML synchronization with AD/LDAP
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When enabled, SAML synchronization with AD/LDAP occurs in phases:
+
+1. Get all the current LDAP users from the Mattermost database who have ``Users.AuthService`` set to ``ldap``. This is a SQL query issued against the Mattermost database.
+2. Get all the current SAML users from the Mattermost database who have ``Users.AuthService`` set to ``saml``. This is a SQL query issued against the Mattermost database.
+3. Get all the current LDAP users from the LDAP server as defined by ``LdapSettings.UserFilter``. This is an LDAP query issued against the LDAP server. Users are retrieved in batches as defined by ``LdapSettings.MaxPageSize``.
```

Comment:
Please review steps 3 - 5 for accuracy. Would it be worth adding the LDAP query to step 3?

Reply:
Maybe add a link to most/mattermost-server/blob/master/scripts/ldap-check.sh for the query